awx/main/tasks.py

```diff
@@ -742,10 +739,29 @@ def get_stdout_handle(self, instance):
         '''
         if not os.path.exists(settings.JOBOUTPUT_ROOT):
             os.makedirs(settings.JOBOUTPUT_ROOT)
-        stdout_filename = os.path.join(settings.JOBOUTPUT_ROOT, "%d-%s.out" % (instance.pk, str(uuid.uuid1())))
+        stdout_filename = os.path.join(
+            settings.JOBOUTPUT_ROOT,
+            "%d-%s.out" % (instance.pk, str(uuid.uuid1()))
+        )
         stdout_handle = codecs.open(stdout_filename, 'w', encoding='utf-8')
```

Comment:
So then is this still writing to the job standard out file? Is getting rid of that on the horizon? Or at least maybe it could be optionally enabled.

Reply:
> Gut all of the "job stdout on the local file system" implementation (we'll lose a lot of complicated code when we do this 😄)

wait, I think that answers my question

Reply:
Yep, this still writes stdout files.  My goal is to gut this, the `unified_job.result_stdout_file` column, and the tall blob of code in the stdout view that contains all the logic for file generating and loading.